host/3.0/buster/amd64/java/java8/java8.Dockerfile

```diff
@@ -48,9 +65,10 @@ ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
 
 COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
 COPY --from=runtime-image [ "/workers/java", "/azure-functions-host/workers/java" ]
-COPY --from=jre [ "/usr/lib/jvm/zre-hl-8-azure-amd64", "/usr/lib/jvm/zre-8-azure-amd64" ]
+COPY --from=runtime-image [ "${JAVA_HOME}", "${JAVA_HOME}"]
```

Comment:
If you do choose to multi-stage the JRE thing we can keep it consistent among all java images.  That is slightly preferable to me.  We could include  a stage in every image that would be the Jre stage.  Then we can copy over whatever the java_home is. 
Would just keep things somewhat consistent between disparate images.